src/io.jl

```diff
@@ -21,13 +22,14 @@ function read_meshfile(filename::String)
     else
       ndims_ = read(attrs(file)["ndim"]) # FIXME once Trixi's 3D branch is merged & released
     end
```

Comment:
Is there any reason this variable is called `ndims_` with a trailing underscore? I just adopted this notation, but I would rather change it to just `ndims` if there is no logical reason to stay with `ndims_`.

Reply:
We have a function `ndims` in Trixi which returns the number of dimensions (this function comes from Base and can also be applied to get the number of dimensions of an array). Thus, it was necessary to call a local variable `ndims_ = ndims(dg)` (or something similar) to not hide the function `ndims`. IIRC, we don't use the function `ndims` here, so it's okay for me to use `ndims` as variable name. What do you think, @sloede?

Reply:
Yep, I just wouldn't want to see ndims used as a variable and a function in the same method. If ndims is not used as a function, calling the variable ndims is OK by me.